File: src/java/cleargl/scenegraph/Camera.java
package cleargl.scenegraph;

import cleargl.GLVector;

public class Camera extends Node {

  protected boolean isTargeted = false;
  protected GLVector target;

  public Camera() {
    this.type = "Camera";
  }

  public void setTarget(GLVector target) {
    this.target = target;
  }

  public boolean isTargeted() {
    return isTargeted;
  }

  public void setTargeted(boolean targeted) {
    isTargeted = targeted;
  }
}

File: src/java/cleargl/scenegraph/Camera.kt
@file:JvmName("Camera")
package cleargl.scenegraph

import cleargl.GLVector

class Camera : Node("Camera") {

    var isTargeted = false
    protected var target: GLVector = GLVector(0.0f, 0.0f, 0.0f)

    init {
        this.nodeType = "Camera"
    }

}